src/js/player.js

```diff
@@ -3650,10 +3650,12 @@ class Player extends Component {
     if (controlBar) {
 
       controlBar.on('mouseenter', function(event) {
-
-        this.player().cache_.inactivityTimeout = this.player().options_.inactivityTimeout;
-        this.player().options_.inactivityTimeout = 0;
-
+        // Fixes bug on Android & iOS where when tapping progressBar (when control bar is displayed)
+        // controlBar would no longer be hidden by default timeout.
+        if (!browser.IS_IOS && !browser.IS_ANDROID) {
```

Comment:
what about moving this check into the the if on line 3650? This way, we don't even add these listeners in the first place

Reply:
Yeah. I think that's a better way to go about it actually. Not sure when i'll have time to make this change though. 

I'll try to get it done by next week :)